breathecode/jobs/tests/actions/tests_get_loc_from_string.py

```diff
@@ -0,0 +1,19 @@
+"""
+Tasks tests
+"""
+from unittest.mock import patch, call
+from ...actions import get_loc_from_string
+from ..mixins import JobsTestCase
+
+
+class ActionGetLocFromStringTestCase(JobsTestCase):
+    def test_get_loc_from_string_is_Empty(self):
+        result = get_loc_from_string('')
+        result_1 = get_loc_from_string('.')
+        result_2 = get_loc_from_string(')')
+        result_3 = get_loc_from_string('(')
```

Comment:
This is not being covered the cade this function return `None` this appear to return three cases